src/pages/ConnectWallet/ConnectWallet.tsx

```diff
@@ -178,83 +157,62 @@ export const ConnectWallet = () => {
-                    </RawText>
-                  </RawText>
-                </Flex>
-                <Text
-                  userSelect={'none'}
-                  color='text.subtle'
-                  fontSize='lg'
-                  textAlign='left'
-                  translation={'connectWalletPage.body'}
-                />
-              </Stack>
-            ) : isSnapEnabled ? (
-              <>
-                <Stack spacing={6}>
-                  <Stack spacing={4} textAlign={textAlign}>
-                    <Heading as='h3' fontSize='2xl'>
-                      {translate('walletProvider.metaMaskSnap.secondaryTitle')}
-                    </Heading>
-                    <RawText color='text.subtle' fontSize='lg'>
-                      {translate('walletProvider.metaMaskSnap.secondaryBody')}
-                    </RawText>
-                    <HStack spacing={4} justify={flexAlign} wrap='wrap' mb={4}>
-                      {renderChains}
-                      <Tooltip label={renderEvmChainText}>
-                        <RawText color='text.subtle'>
-                          {translate('walletProvider.metaMaskSnap.andMore')}
-                        </RawText>
-                      </Tooltip>
-                    </HStack>
-                  </Stack>
-                  <Button
-                    width='full'
-                    size='lg-multiline'
-                    onClick={handleMetaMaskConnect}
-                    leftIcon={metamaskIcon}
-                  >
-                    {translate('walletProvider.metaMaskSnap.connectMetaMask')}
-                  </Button>
-                </Stack>
-                <Flex alignItems='center' justifyContent='center' width='full' gap={4}>
-                  <Divider flex={1} borderColor='border.bold' opacity='1' />
-                  <Text
-                    color='text.subtle'
-                    fontWeight='medium'
-                    textAlign='center'
-                    translation='common.or'
-                  />
-                  <Divider flex={1} borderColor='border.bold' opacity='1' />
-                </Flex>
-              </>
-            ) : (
-              <Stack spacing={6}>
-                <Heading as='h3' fontSize='2xl'>
-                  {translate('connectWalletPage.welcomeBack')}
+            <Stack spacing={6}>
+              <Stack spacing={4} textAlign='center' alignItems='center' fontWeight='bold'>
+                <RawText color='blue.300' fontSize='m' textTransform='uppercase'>
+                  {translate('connectWalletPage.secondaryTitle')}
+                </RawText>
+                <Heading as='h1' fontSize='4xl'>
+                  {translate('connectWalletPage.primaryTitle')}
                 </Heading>
-                <RawText fontSize='lg' color='text.subtle'>
-                  {translate('connectWalletPage.welcomeBody')}
+                <RawText color='text.subtle' fontSize='m'>
```

Comment:
```suggestion
                <RawText color='text.subtle' fontSize='md' fontWeight="medium">
```

Reply:
Addressed in https://github.com/shapeshift/web/pull/7793/commits/09dc620fb5154c0de092a12416e1bcdcbf403e64